### PR TITLE
Add column "Created"

### PR DESCRIPTION
**Pull request by strazto**
## Related Issue

Saw it discussed in https://github.com/jesec/flood/discussions/360 

## Types of changes

- [ ] Breaking change (changes that break backward compatibility of public API or CLI - semver MAJOR)
- [x] New feature (non-breaking change which adds functionality - semver MINOR)
- [ ] Bug fix (non-breaking change which fixes an issue - semver PATCH)

### CODE REVIEW

**Comment by matthewstrasiotto** [comment]
@jesec - Would you mind reviewing?

**Comment by matthewstrasiotto** [comment]
https://github.com/jesec/flood/pull/414#issuecomment-929833605

> There is a minor issue: don't touch the translations. They are handled by Crowdin. Only change en.json. 

I'm glad to hear that's the expectation, since that's all I did - do you think you could document this in the contributor guidelines?

**Comment by jesec** [comment]
> [#414 (comment)](https://github.com/jesec/flood/pull/414#issuecomment-929833605)
> 
> > There is a minor issue: don't touch the translations. They are handled by Crowdin. Only change en.json.
> 
> I'm glad to hear that's the expectation, since that's all I did - do you think you could document this in the contributor guidelines?

I don't understand how that relates to this PR.

File: shared/constants/defaultFloodSettings.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ const defaultFloodSettings: Readonly<FloodSettings> = {
     {id: 'seeds', visible: true},
     {id: 'dateAdded', visible: true},
     {id: 'dateCreated', visible: false},
+    {id: 'dateFinished', visible: true},
```
[the remark this answers is not in the document]
Comment (matthewstrasiotto) [reply shared/constants/defaultFloodSettings.ts:23]: Up to you

**Comment by matthewstrasiotto** [comment]
> > [#414 (comment)](https://github.com/jesec/flood/pull/414#issuecomment-929833605)
> > > There is a minor issue: don't touch the translations. They are handled by Crowdin. Only change en.json.
> > 
> > 
> > I'm glad to hear that's the expectation, since that's all I did - do you think you could document this in the contributor guidelines?
> 
> I don't understand how that relates to this PR.

I found myself confused about what was expected of me for the translations, I found it on another PR, and I think it could be documented somewhere in some contributing guidelines

File: shared/constants/defaultFloodSettings.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthewstrasiotto) [reply shared/constants/defaultFloodSettings.ts:23]: I'm guessing you don't want it displayed by default, so I'll change it.

**Pull request closed without merging**